modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -11,60 +11,45 @@
 .Default CNI network plugin feature comparison
 [cols="50%,25%,25%",options="header"]
 |===
-ifeval::["{context}" == "about-ovn-kubernetes"]
-|Feature|OVN-Kubernetes|OpenShift SDN
 
-|Egress IPs|Supported|Supported
-
-|Egress firewall ^[1]^|Supported|Supported
-
-|Egress router|Supported ^[2]^|Supported
-
-|Hybrid networking|Supported|Not supported
-
-|IPsec encryption for intra-cluster communication|Supported|Not supported
-
-|IPv6|Supported ^[3]^ ^[4]^|Not supported
-
-|Kubernetes network policy|Supported|Supported
-
-|Kubernetes network policy logs|Supported|Not supported
-
-|Hardware offloading|Supported|Not supported
-
-|Multicast|Supported|Supported
-endif::[]
-ifeval::["{context}" == "about-openshift-sdn"]
 |Feature|OpenShift SDN|OVN-Kubernetes
 
 |Egress IPs|Supported|Supported
 
-|Egress firewall ^[1]^|Supported|Supported
+|Egress firewall|Supported|Supported ^[1]^
 
 |Egress router|Supported|Supported ^[2]^
 
 |Hybrid networking|Not supported|Supported
 
-|IPsec encryption|Not supported|Supported
+|IPsec encryption for intra-cluster communication|Not supported|Supported
+
+|IPv4 single-stack|Supported|Supported
+
+|IPv6 single-stack|Not supported|Supported ^[3]^
+
+|IPv4/IPv6 dual-stack|Not Supported|Supported ^[4]^
 
-|IPv6|Not supported|Supported ^[3]^ ^[4]^
+|IPv6/IPv4 dual-stack|Not supported|Supported ^[5]^
 
 |Kubernetes network policy|Supported|Supported
 
 |Kubernetes network policy logs|Not supported|Supported
 
+|Hardware offloading|Not supported|Supported
+
 |Multicast|Supported|Supported
 
-|Hardware offloading|Not supported|Supported
-endif::[]
 |===
 [.small]
 --
 1. Egress firewall is also known as egress network policy in OpenShift SDN. This is not the same as network policy egress.
 
 2. Egress router for OVN-Kubernetes supports only redirect mode.
 
-3. IPv6 is supported only on bare metal, {ibmpowerProductName}, and {ibmzProductName} clusters.
+3. IPv6 single-stack networking on a bare-metal platform.
+
+4. IPv4/IPv6 dual-stack networking on bare-metal, {vmw-full} (installer-provisioned infrastructure installations only), {ibm-power-name}, and {ibm-z-name} platforms.
 
-4. IPv6 single stack does not support xref:../../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Kubernetes NMState] and is not supported on {ibmpowerProductName} and {ibmzProductName} clusters.
+5. IPv6/IPv4 dual-stack networking on bare-metal, {vmw-full} (installer-provisioned infrastructure installations only), and {ibm-power-name} platforms.
```

Comment:
4.13 vSphere does not support IPv6/IPv4.

Reply:
Thanks. Updated.